samples/msal-angular-v2-samples/angular-b2c-sample-app/karma.conf.js

```diff
@@ -13,10 +13,19 @@ module.exports = function (config) {
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client: {
+      jasmine: {
+        // you can add configuration options for Jasmine here
+        // the possible options are listed at https://jasmine.github.io/api/edge/Configuration.html
+        // for example, you can disable the random execution with `random: false`
+        // or set a specific seed with `seed: 4321`
+      },
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
+    jasmineHtmlReporter: {
+      suppressAll: true // removes the duplicated traces
+    },
     coverageReporter: {
-      dir: require('path').join(__dirname, './coverage/angular11-sample-app'),
+      dir: require('path').join(__dirname, './coverage/angular14-b2c-sample'),
```

Comment:
Does this need to be changed to `angular-b2c-sample-app`?

Reply:
good catch! will fix